docs/user/FITBIT_EXPORT.md

```
@@ -0,0 +1,123 @@
+
+After unzipping, copy the relevant folders/files into:
+
+- `data/input/fitbit/`
+
+---
+
+## 2) Required Folder Layout
+
+Expected structure:
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)